self-service-kiosk/src/features/counter/Counter.module.css

```diff
@@ -0,0 +1,78 @@
+.row {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
+
+.row > button {
+  margin-left: 4px;
```

Comment:
The size should not be in pixels so that the user interface is responsive. As this is an example, it is not very important, but it is something to watch for in the future.